routes/controllers/postController.ts

```diff
@@ -3,10 +3,31 @@ import { Router } from 'express';
 import verifyToken from '../middleware/jwt/verifyToken';
 import postValidation from '../middleware/post/_validation';
 import checkValidation from '../middleware/common/checkValidation';
-import postCreate from '../middleware/post/createPost';
+import checkUser from '../middleware/post/common/checkUser';
+
+//createPost
+import createPost from '../middleware/post/createPost';
+
+//editPost
+import editPost from '../middleware/post/editPost';
+
+//deletePost
+import deletePost from '../middleware/post/deletePost';
+
+//likePost
+import likePost from '../middleware/post/likePost';
+
+//comment
+import createComment from '../middleware/post/comment/createComment';
+import commentValidation from '../middleware/post/comment/_validation';
 
 const router = Router();
 
```

Comment:
```suggestion
router.use(verifyToken)
```
verifyToken 미들웨어가 모든 해당 컨트롤러에서 다 사용된다면 해당 코드를 추천